docker-compose.yml

```diff
@@ -28,9 +30,22 @@ services:
     build: ./
     environment:
       - CELERY_BROKER_URL=amqp://guest@rabbitmq//
-    command: celery worker -A AIPscan.worker.celery --loglevel=info
+      - TYPESENSE_HOST=typesense
+      - TYPESENSE_API_KEY=xyz
+    command: celery -A AIPscan.worker.celery worker --loglevel=info
     volumes:
       - "./:/src:rw"
     depends_on:
       - rabbitmq
       - aipscan
+
+  typesense:
+    image: typesense/typesense:0.26.0.rc23
+    environment:
+      TYPESENSE_DATA_DIR: /data
+      TYPESENSE_API_KEY: xyz
+    volumes:
+      - /tmp/typesense-server-data:/data
+    ports:
+      - 8108:8108
+    restart: "no"
```

Comment:
Why?

Reply:
Yeah, this is unneeded and I'll remove. Thanks!